src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLoggerProvider.cs

```diff
@@ -60,8 +60,14 @@ public ConsoleLoggerProvider(IOptionsMonitor<ConsoleLoggerOptions> options, IEnu
             }
         }
 
+        [UnsupportedOSPlatformGuard("windows")]
         private static bool DoesWindowsConsoleSupportAnsi()
         {
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
This is just reverting the old [behavior](https://github.com/dotnet/runtime/pull/50931/files#diff-179c35fce8da76fa9e41abb6cd536c276552568d14f535f5ad4cbf5d0515e626L64-L67) and using guard annotation instead

Reply:
The method name should be reverted to `DoesConsoleSupportAnsi`